CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+**Contributions are welcome.**
+
+## Contribution guidelines
+
+General [RIOT contribution guidelines](https://github.com/RIOT-OS/RIOT/blob/master/CONTRIBUTING.md) apply,
+with some
+
+* focus points:
+
+    * When wrapping a new RIOT feature, **keep it small**.
+      Attempting to implement all features of a particular RIOT subsystem easily gets caught in long iterations of review.
+      Try wrapping a minimal viable version first, and let's take it from there.
+
```

Comment:
If you like PRs to be submitted as draft, you could add "PRs can be submitted as work-in-progress by marking them as draft" here.

(Or add this to the general RIOT contributing guide)